x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/job/DataFrameJobState.java

```diff
@@ -30,7 +31,7 @@
 import static org.elasticsearch.common.xcontent.ConstructingObjectParser.optionalConstructorArg;
 
 public class DataFrameJobState implements Task.Status, PersistentTaskState {
-    public static final String NAME = "xpack/data_frame/job_state";
+    public static final String NAME = DataFrame.TASK_NAME;
```

Comment:
There is a utility class `MlTasks` where we've defined these constants for jobs and datafeeds

Reply:
+1, I plan to de-dup all those fields and strings and create a utility class as well, in a separate PR